examples/EvaluationDataToApplicationInsights/appsettings.json

```diff
@@ -0,0 +1,59 @@
+{
+    "Logging": {
+        "LogLevel": {
+            "Default": "Information",
+            "Microsoft.AspNetCore": "Warning"
+        }
+    },
+    "AllowedHosts": "*",
+    "ApplicationInsights": {
+    },
+    "FeatureManagement": {
+        "ImageRating": {
+            "TelemetryEnabled": true,
+            "Variants": [
+                {
+                    "Name": "BlackAndWhite",
+                    "ConfigurationValue": "https://cdn.pixabay.com/photo/2017/09/28/09/26/dinosaur-2794840_1280.png"
```

Comment:
Is this something we should worry about copyright?

Reply:
No, I made sure to use royalty free images. pixabay is free to use in any way. From their site.

"All content is released by Pixabay under the Content License, which makes it safe to use without asking for permission or giving credit to the artist - even for certain commercial purposes."

Reply:
Perfect. Thanks!